src/games/_gameswidget.py

```diff
@@ -14,10 +15,17 @@
 import notifications as ns
 
 import logging
+
+SORT_GAMES = "play/sortGames"
+HIDE_PRIVATE_GAMES = "play/hidePrivateGames"
+
 logger = logging.getLogger(__name__)
 
 FormClass, BaseClass = util.loadUiType("games/games.ui")
 
+Settings.persisted_property(HIDE_PRIVATE_GAMES, default_value=False, type=bool)
+Settings.persisted_property(SORT_GAMES, default_value=0, type=int)
```

Comment:
This is meant to be used as a property on a class, calling these functions like this initializes a setter/getter but then immediately discards it -- that doesn't seem right.

You'll want to assign it on the GamesWidget class like so:

``` class GamesWidget(FormClass, BaseClass):
...
    sort_games = Settings.persisted_property(...)```

After which `self.sort_games = index` will automatically persist it, and `self.sort_games` will yield the value from configuration.